_includes/patterns-filter.html

```diff
@@ -279,11 +281,21 @@
       }
   }
 
+  // This calculates the count of each technology and stores it in a object which looks like a key value pair
+  // example: {"deployment": 102, "security": 10 ...} and so on 
+  const categoryCardCount = Array.from(patternCards).reduce((acc, singleCard) => {
+    const type = singleCard.getAttribute("type").toLowerCase();
+    acc[type] = (acc[type] || 0) + 1;
+    return acc;
+  }, {});
```

Comment:
@kanishk6103 is this necessary, can we not add this logic in same code that identify count for technology I think this will be unnecessary loop because we already doing it for technology, at same time we can identify category

Reply:
Sure, I can do that i believe.